qutebrowser/browser/webengine/webview.py

```diff
@@ -144,6 +146,18 @@ def _set_bg_color(self):
             col = self._theme_color
         self.setBackgroundColor(col)
 
+    @pyqtSlot(QUrl, 'QAuthenticator*', 'QString')
+    def _on_proxy_authentication_required(self, url, authenticator, proxyHost):
```

Comment:
`proxyHost` should be `proxy_host` as per [pep8](http://pep8.org/), or `_proxy_host` if unused (but see below)